frontend/src/app/testQueue/TestQueue.test.tsx

```diff
@@ -178,7 +178,7 @@ describe("TestQueue", () => {
     expect(
       screen.getByText("To add results in bulk", { exact: false })
     ).toHaveTextContent(
-      "To add results in bulk using a CSV file, go to Upload spreadsheet."
+      "To add results in bulk for COVID-19, Flu A and B, and RSV using a CSV file, go to Upload spreadsheet."
```

Comment:
could we pull this string snippet for "all diseases we support" into a constant and then use that in the few places in the app where we need to list out all the diseases? Might make the next version of this change for once we launch new diseases simpler.

Reply:
I'll add a follow up PR to have [this other ticket PR](https://github.com/CDCgov/prime-simplereport/pull/7369) use the same constant as well